DB/src/test/java/io/deephaven/db/v2/LiveTableTestCase.java

```diff
@@ -30,33 +32,51 @@
 abstract public class LiveTableTestCase extends BaseArrayTestCase implements UpdateErrorReporter {
     static public boolean printTableUpdates = Configuration.getInstance()
             .getBooleanForClassWithDefault(LiveTableTestCase.class, "printTableUpdates", false);
+    private static final boolean ENABLE_COMPILER_TOOLS_LOGGING = Configuration.getInstance()
+            .getBooleanForClassWithDefault(QueryTableTestBase.class, "CompilerTools.logEnabled", false);
 
     private boolean oldMemoize;
     private UpdateErrorReporter oldReporter;
     private boolean expectError = false;
     private SafeCloseable scopeCloseable;
+    private QueryScope originalScope;
```

Comment:
Maybe call this `originalQueryScope` and rename `scopeCloseable` to `livenessScopeCloseable` for clarity?